tests/Aspire.Hosting.Docker.Tests/DockerComposePublisherTests.cs

```diff
@@ -65,7 +73,7 @@ public async Task PublishAsync_GeneratesValidDockerComposeFile()
             """
             services:
             myapp:
-                image: "mcr.microsoft.com/dotnet/aspnet:8.0"
+                image: "${MYAPP_IMAGE}"
```

Comment:
I don't like this change. This is a fixed container image, not something that needs to change per environment.

Reply:
Rolling back.